docs/explanations/modeling_extensions/surrogate/api/omlt-keras/index.rst

```diff
@@ -1,10 +1,71 @@
 OMLT: Optimization and Machine Learning Toolkit
 ===============================================
 
-Keras is a deep learning framework that integrates with TensorFlow's structure for
-building and training artificial neural networks, and minimizes the number of user
-actions required to construct accurate networks. OMLT (Optimization and Machine
-Learning Toolkit) provides an interface to formulate machine learning models and
-import Keras or ONNX models as Pyomo blocks. The provided tools include an interface
-for accessing the Keras module (via the publicly available Python package) within
-IDAES flowsheets.
\ No newline at end of file
+.. toctree::
+    :maxdepth: 1
+
+    omlt-keras-options
+
+Keras is a deep learning framework that integrates with TensorFlow's structure for building and training artificial neural networks, and minimizes the number of user actions required to construct accurate networks. OMLT (Optimization and Machine Learning Toolkit) provides an interface to formulate machine learning models and import Keras or ONNX models as Pyomo blocks. The provided tools include an interface for accessing the Keras module (via the publicly available Python package) within IDAES flowsheets.
+
+Basic Usage
+-----------
+
+Keras-OMLT main function is **keras_surrogate.KerasSurrogate**, which populates an IDAES `SurrogateObject` with the OMLT model. This object may then be passed directly to other IDAES methods for visualization or flowsheet integration (see the sections for Visualization and Examples below).
+
+Data can be read in or simulated using available Python packages. The main arguments of the `keras_surrogate.KerasSurrogate`` Python function are inputs and outputs. For example,
+
+.. code-block:: python
+
+  # selected settings for regression
+  
+  activation, optimizer, n_hidden_layers, n_nodes_per_layer = "tanh", "Adam", 2, 40
+  loss, metrics = "mse", ["mae", "mse"]  
+
+  # Create data objects for training using scalar normalization
+  model = tf.keras.Sequential()
+  model.add(
+      tf.keras.layers.Dense(units=n_nodes_per_layer, input_dim=len(input_labels), activation=activation))
+  for i in range(1, n_hidden_layers):
```

Comment:
I'd suggest adding another in-line comment calling out that we are creating hidden layers.